packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -27,10 +27,41 @@ export interface RadioGroupState extends RadioGroupProps {
    */
   selectedButtonRef?: React.MutableRefObject<any>;
 
+  /**
+   * Populate the values arrays
+   */
+  addRadioValue?: (value: string) => void;
+  removeRadioValue?: (value: string) => void;
+
   /**
    * Array of radio keys in the group
    */
   values?: string[];
+
+  /**
+   * Populate the enabledValues arrays
+   * @platform win32
+   */
+  addRadioEnabledValue?: (value: string) => void;
```

Comment:
What's the difference between `addRadioEnabledValue()` and `addRadioValue()`? I'd like the comments to be a little more in-depth

Reply:
`addRadioEnabledValue()` adds the Radio key to the `enabledValues` array and `addRadioValue()` adds the Radio key to the values array. Updated the description!